acct-user/aerospike-amc-community/aerospike-amc-community-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Aerospike-amc-community acct-user"
+
+ACCT_USER_ID=96
```

Comment:
New users/groups should used a higher uid/gid. See `used_free_uidgids.sh`.
`Recommended UID+GID pair: 211`

Reply:
Also please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files